recipe/meta.yaml

```
@@ -22,14 +25,80 @@ requirements:
   host:
     - ncurses
 
-test:
-  commands:
-    - test -f $PREFIX/lib/libestbase.so  # [linux]
-    - test -f $PREFIX/lib/libestbase.so.2.5  # [linux]
-    - test -f $PREFIX/lib/libestools.so  # [linux]
-    - test -f $PREFIX/lib/libestools.so.2.5  # [linux]
-    - test -f $PREFIX/lib/libeststring.so  # [linux]
-    - test -f $PREFIX/lib/libeststring.so.2.5  # [linux]
+outputs:
+  - name: libestools
+    script: install.sh  # [linux]
+    build:
+      run_exports:
+        - {{ pin_subpackage('libestools', max_pin='x.x') }}
+
+    test:
+      commands:
+        {% set files = [
+          "libestbase.so." + version, "libestools.so." + version,
+          "libeststring.so." + version,
+        ] %}
+        {% for f in files %}
+        - test -f $PREFIX/lib/{{ f }}  # [linux]
+        {% endfor %}
+
+        {% set links = [
+          "libestbase.so." + ver2, "libestools.so." + ver2,
+          "libeststring.so." + ver2,
+        ] %}
+        {% for l in links %}
+        - test -L $PREFIX/lib/{{ l }}  # [linux]
+        - test -e $PREFIX/lib/{{ l }}  # [linux]
+        {% endfor %}
+
+  - name: libestools-dev
+    script: install_devel.sh  # [linux]
+    build:
+      run_exports:
+        - {{ pin_subpackage('libestools', max_pin='x.x') }}
+
+    requirements:
+      run:
+        - {{ pin_subpackage("libestools", exact=True) }}
+    test:
+      commands:
+        {% set files = [
+          "include/speech_tools/EST.h", "lib/speech_tools/config/config",
+          "lib/speech_tools/lib/siod/init.scm",
+        ] %}
+        {% for f in files %}
+        - test -f $PREFIX/{{ f }}  # [linux]
+        {% endfor %}
+
+        {% set links = [
+          "lib/libestbase.so", "lib/libestools.so", "lib/libeststring.so",
+        ] %}
+        {% for l in links %}
+        - test -L $PREFIX/{{ l }}  # [linux]
+        - test -e $PREFIX/{{ l }}  # [linux]
+        {% endfor %}
+
+  - name: speech-tools
+    script: install_bin.sh  # [linux]
+    build:
+      run_exports:
+        - {{ pin_subpackage('libestools', max_pin='x.x') }}
+
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - libgomp  # [linux]
+      run:
+        - {{ pin_subpackage("libestools", max_pin='x.x') }}
+    test:
```

```suggestion
    requirements:
      build:
        - {{ compiler('c') }}
        - {{ compiler('cxx') }}
        - libgomp  # [linux]
      host:
        - {{ pin_subpackage("libestools", max_pin='x.x') }}
      run:
        - {{ pin_subpackage("libestools", max_pin='x.x') }}
    test:
```
I was slightly wrong. Specific pinning is needed, but also the pinning in host is still needed so that conda link checking works.